### PR TITLE
[IMP] inventory: troubleshooting 3rd party carrier

### PR DESCRIPTION
**Pull request by Felicious**
Add troubleshooting section to help users struggling to set up shipping connectors

Merge after https://github.com/odoo/documentation/pull/8634

[Task](https://www.odoo.com/web#id=3873888&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

Fwport: yes

### CODE REVIEW

**Comment by brse-odoo** [comment]
@Felicious I'll review this right away!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (brse-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:11]:
```suggestion
In Odoo, shipping carriers can be applied to sales orders (SOs), invoices, or delivery orders. For
```
If it's multiple sales orders, should we add an -s here? I know it looks a bit weird so it's up to you

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (brse-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:255]:
```suggestion
   (e.g., address and phone number) in Odoo is correct **and** match the records saved in the
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (brse-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:268]:
```suggestion
To track inconsistencies in the shipping data between Odoo and the shipping carrier, activate the debug log.
```

**Review by brse-odoo — APPROVED** [review]
@Felicious I've finished my review of this PR, great job! I'm approving with very few suggestions for you to accept/reject as you see fit. Let me know if you have any questions, thanks!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:287]:
```suggestion
   (e.g. address and phone number) in Odoo is correct **and** match the records saved in the
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:302]:
```suggestion
      six kilograms, but the response from FedEx says the package weights seven kilograms, it
      concludes that the issue is on FedEx's side.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:318]:
```suggestion
:ref:`developer mode <developer-mode>`, and go to :menuselection:`Settings app --> Technical -->
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:319]:
```suggestion
Database Structure section --> Logging`.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:322]:
```suggestion
   Logs are created for a shipping method each time the :ref:`Get Rate
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:331]:
```suggestion
Click the *HTTP request* line item to open a detailed page, and verify the correct information is
```

**Review by ksc-odoo — APPROVED** [review]
Hey @Felicious -- just finished my Final Review. Nice work! After you address the few comments I left, feel free to tag this for Tech Review. Approving now! 👍

**Comment by Felicious** [comment]
Hello @samueljlieber, this PR is ready for your technical review! This is meant to merge after https://github.com/odoo/documentation/pull/8634/

Thank you so much, as usual :D

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:16]: Adding a little more meaning into the link label, so it is clear to the user where they will end up after clicking the link :)
```suggestion
tips on resolving common issues when configuring shipping connectors, skip to the :ref:`Troubleshooting
<inventory/shipping_receiving/third-party-troubles>`  section.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:287]: Should this be "matches"?
```suggestion
   (e.g., address and phone number) in Odoo is correct **and** matches the records saved in the
```

**Review by samueljlieber — APPROVED** [review]
Hi @Felicious, your improvements to this doc look good to me! I have two quick comments, but no corrections. Nice job! 

Please tag me for one more look after rebasing this onto base branch once https://github.com/odoo/documentation/pull/8634 is merged and I will quickly verify it is ready to go. Thank you!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:16]: This is really good advice! I'll make sure all the link labels are named, from now on!

**Comment by Felicious** [comment]
Hi @samueljlieber ! The PR that occurs before this has merged, so I changed the target of the PR to 16.0. It's ready for you to take one last look before merging! :D

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:16]: Nitpick about meaningful link names, just so its clear to the reader which section you are referring to :)
```suggestion
tips on resolving common issues when configuring shipping connectors, skip to the
:ref:`Troubleshooting <inventory/shipping_receiving/third-party-troubles>` section.
```

**Review by samueljlieber — APPROVED** [review]
Nice PR @Felicious! Your changes look good to me, I have a single small nitpick but other than that no changes! Delegating merge to you and approving 👍
..
@robodoo delegate=Felicious

**Comment by Felicious** [comment]
@robodoo r+

**Pull request closed without merging**
